core/modules/field/modules/options/css/options.css

```diff
@@ -0,0 +1,102 @@
+
+div.options-widget .draggable a.tabledrag-handle {
```

Comment:
Throughout this entire file we should remove all _possible_ tag selectors like `div`, `table`, `td`, etc. to make it easier for themers to use less-specific targeting. Some of these tags will probably be necessary to retain, but be as generic as possible.

Reply:
I've removed a bunch of these. Though I left some in to more closely match how field_ui.css does it (such as `a`, `td`)